backend/controllers/caseNote.ts

```diff
@@ -203,37 +211,8 @@ export const caseNoteFactory = (prisonApi, caseNotesApi) => {
     }
 
     if (errors.length > 0) {
-      const { firstName, lastName } = await prisonApi.getDetails(res.locals, offenderNo)
-
-      const caseNoteTypes = await caseNotesApi.myCaseNoteTypes(res.locals)
-
-      const types = caseNoteTypes.map((caseNoteType) => ({
-        value: caseNoteType.code,
-        text: caseNoteType.description,
-      }))
-
-      const subTypes = caseNoteTypes
-        .map((caseNoteType) =>
-          caseNoteType.subCodes.map((subCode) => ({
-            type: caseNoteType.code,
-            value: subCode.code,
-            text: subCode.description,
-          }))
-        )
-        .reduce((result, subCodes) => result.concat(subCodes), [])
-        .filter((sub) => (type === undefined ? true : sub.type === type))
-
-      if (req.xhr) {
-        const { typeCode } = req.query
```

Comment:
think this `xhr` stuff was just a copy and paste from the `index` path since we won't get an `xhr` `post` request anyway :shrug: